### PR TITLE
write unit test for DeleteOrganizationHandlerAsync #786

### PR DESCRIPTION
**Pull request by ankurMalhotra**

### CODE REVIEW

**Comment by tonysurma** [comment]
@mgmccarthy quick review before we merge?

@ankurMalhotra thanks for contributing!  greatly appreciated.

**Comment by mgmccarthy** [comment]
@tonysurma, dev machine is smoked, until I get it up and running, so this will be an "eyeball" code review

**Comment by tonysurma** [comment]
@mgmccarthy no problem!

File: AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Features/Organizations/DeleteOrganisationHandlerTests.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
+﻿using AllReady.Areas.Admin.Features.Organizations;
+using AllReady.Models;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace AllReady.UnitTest.Areas.Admin.Features.Organizations
+{
+    public class DeleteOrganizationHandlerAsyncTests : InMemoryContextTest
```
Comment (mgmccarthy) [inline AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Features/Organizations/DeleteOrganisationHandlerTests.cs]: please rename the test to DeleteOrganizationHandlerAsyncShould.

File: AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Features/Organizations/DeleteOrganisationHandlerTests.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        protected override void LoadTestData()
+        {
+            Context.Organizations.Add(new Organization { Id = 1, Name = "Org 1" });
+            Context.Organizations.Add(new Organization { Id = 2, Name = "Org 2" });
+            Context.SaveChanges();
+        }
+
+        [Fact]
+        public async Task DeleteExistingOrganization()
+        {
```
Comment (mgmccarthy) [inline AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Features/Organizations/DeleteOrganisationHandlerTests.cs]: DeleteOrganizationThatMatchesOrganizationIdOnCommand

File: AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Features/Organizations/DeleteOrganisationHandlerTests.cs [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            var command = new DeleteOrganizationAsync { Id = 1 };
+            var handler = new DeleteOrganizationHandlerAsync(Context);
+            await handler.Handle(command);
+            var data = Context.Organizations.Count(_ => _.Id == 1);
+            Assert.Equal(0, data);
+        }
+
+        [Fact]
+        public async Task DeleteNonExistantOrganization()
```
Comment (mgmccarthy) [inline AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Features/Organizations/DeleteOrganisationHandlerTests.cs]: rename to `NotDeleteOrganizationsThatDoNotMatchOrganizationIdOnCommand`

**Comment by mgmccarthy** [comment]
@tonysurma, after the re-namings are made above :shipit:

**Comment by mgmccarthy** [comment]
@tonysurma, my dev machine is toast. I have to wait until next Tuesday to get it fixed by our comapany's service desk, so I will be unable to do any code reviews until then

**Pull request merged**